checkov/terraform/checks/resource/base_resource_value_check.py

```diff
@@ -94,3 +95,6 @@ def get_expected_value(self):
         Returns the default expected value, governed by provider best practices
         """
         return True
+
+    def get_evaluated_keys(self):
```

Comment:
Is this necessary? I think it should pick the function up from base_check.

Reply:
They don't get the same values, here it takes `get_inspected_key()` and in base_check it takes `self.evaluated_keys`.
Anyway @nimrodkor this is relevant to my question in the first comment

Reply:
I think this is a nice shortcut to keep the implementation of `BaseResourceValueCheck` as simple as it gets. We know what the attribute is - no need to look for it and generate it in the scan